includes/admin/class-wc-stripe-payment-requests-controller.php

```diff
@@ -19,46 +19,41 @@ public function __construct() {
 	 */
 	public function admin_scripts() {
 		// Webpack generates an assets file containing a dependencies array for our built JS file.
-		$script_path       = 'build/payment_requests_customizer.js';
-		$script_asset_path = WC_STRIPE_PLUGIN_PATH . '/build/payment_requests_customizer.asset.php';
-		$script_url        = plugins_url( $script_path, WC_STRIPE_MAIN_FILE );
-		$script_asset      = file_exists( $script_asset_path )
+		$script_asset_path = WC_STRIPE_PLUGIN_PATH . '/build/payment_requests_settings.asset.php';
+		$asset_metadata    = file_exists( $script_asset_path )
 			? require $script_asset_path
 			: [
 				'dependencies' => [],
 				'version'      => WC_STRIPE_VERSION,
 			];
 		wp_register_script(
-			'wc_stripe-payment-requests_customizer',
-			$script_url,
-			$script_asset['dependencies'],
-			$script_asset['version'],
+			'wc-stripe-payment-request-settings',
+			plugins_url( 'build/payment_requests_settings.js', WC_STRIPE_MAIN_FILE ),
+			$asset_metadata['dependencies'],
+			$asset_metadata['version'],
 			true
 		);
-
-		wp_enqueue_script( 'wc_stripe-payment-requests_customizer' );
+		wp_enqueue_script( 'wc-stripe-payment-request-settings' );
 
 		wp_register_style(
-			'wc_stripe-payment-requests_customizer',
-			plugins_url( 'build/payment_requests_customizer.css', WC_STRIPE_MAIN_FILE ),
-			[],
-			$script_asset['version']
+			'wc-stripe-payment-request-settings',
+			plugins_url( 'build/payment_requests_settings.css', WC_STRIPE_MAIN_FILE ),
+			[ 'wc-components' ],
```

Comment:
oops, looks like I missed this dependency in my original PR. What tipped you off that you needed it?

Reply:
Nothing much - just covering our bases. I think they're already queued by other scripts in the page, but I just wanted to explicitly declared them here as well